x-pack/plugins/ml/server/models/job_audit_messages/job_audit_messages.ts

```diff
@@ -400,9 +407,72 @@ export function jobAuditMessagesProvider(
     return (Object.keys(LEVEL) as LevelName[])[Object.values(LEVEL).indexOf(level)];
   }
 
+  /**
+   * Retrieve list of errors per job.
+   * @param jobIds
+   */
+  async function getJobsErrors(jobIds: string[], earliestMs?: number): Promise<JobsErrorsResponse> {
+    const { body } = await asInternalUser.search({
+      index: ML_NOTIFICATION_INDEX_PATTERN,
+      ignore_unavailable: true,
+      size: 0,
+      body: {
+        query: {
+          bool: {
+            filter: [
+              ...(earliestMs ? [{ range: { timestamp: { gte: earliestMs } } }] : []),
+              { terms: { job_id: jobIds } },
+              {
+                term: { level: { value: 'error' } },
```

Comment:
If the comment above about making message level a param isn't added, this should be `MESSAGE_LEVEL.ERROR`

Reply:
Changed in https://github.com/elastic/kibana/pull/108701/commits/54cc87ae63b42b5de34fbae4a594987e5a11112f